requirements.txt

```diff
@@ -16,3 +16,7 @@ geojson==2.5.0
 python-dotenv==0.15.0
 # memory profiling
 guppy3==3.1.0
+# read xlsx files
+openpyxl==3.0.7
+# read xls files
+xlrd==2.0.1
```

Comment:
Do we use these?

Reply:
If we remove them, `file_to_json` will throw the following:

`ImportError: Missing optional dependency 'xlrd'. Install xlrd >= 1.0.0 for Excel support Use pip or conda to install xlrd.`

After pip installing that, `xlrd` will throw:

`xlrd.biffh.XLRDError: Excel xlsx file; not supported`

So we need to install `openpyxl` for xlsx support.